src/multi_physics/QED/CMakeLists.txt

```
@@ -1,19 +1,37 @@
-cmake_minimum_required(VERSION 3.5)
+add_library(QED INTERFACE)
+add_library(${PROJECT_NAME}::QED ALIAS QED)
 
-set(CMAKE_CXX_EXTENSIONS OFF)
+target_compile_features(QED INTERFACE cxx_std_14)
```

Let us try this, because this might be changing the language of the files:
```suggestion
if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
    target_compile_features(QED INTERFACE cxx_std_14)
endif()
```